ctapipe/containers.py

```diff
@@ -104,8 +107,10 @@ class ObservingMode(enum.Enum):
 
 class PointingMode(enum.Enum):
     UNKNOWN = -1
-    TRACK = 0  # track a fixed point that moves with the sky
-    DRIFT = 1  # track a fixed alt/az point
+    #: drives track a fixed point that moves with the sky
```

Comment:
A "fixed point" that moves sounds a bit strange, although all in all, it is clear what is meant.

Maybe: "Drives track a fixed position in ICRS".